go/vt/vttablet/tabletserver/health_streamer.go

```diff
@@ -323,8 +326,8 @@ func (hs *healthStreamer) reload() error {
 		}
 	}
 
-	// TODO: fix the maxrows from using a magic number.
-	qr, err := conn.Exec(ctx, mysql.DetectSchemaChange, 10000, false)
+	maxrows := 10000
+	qr, err := conn.Exec(ctx, mysql.DetectSchemaChange, maxrows, false)
```

Comment:
I meant we should, stream the results than having maxrows

Reply:
Why do we want to stream? I don't follow the logic